Header header
std_msgs/Float64 expected
std_msgs/Float64 threshold
std_msgs/Float64 state_1
std_msgs/Float64 state_2
std_msgs/Float64 state_3
std_msgs/Float64 state_4
